dask_gdf/accessor.py

```diff
@@ -83,10 +83,37 @@ def __getattr__(self, key):
         else:
             raise AttributeError(key)
 
+# Stolen from
+# https://github.com/pandas-dev/pandas/blob/master/pandas/core/accessor.py
```

Comment:
This is not the appropriate way to list copyright.  Please follow [the pandas license](https://github.com/pandas-dev/pandas/blob/master/LICENSE).  Namely you need to include the text:

> Copyright (c) 2008-2012, AQR Capital Management, LLC, Lambda Foundry, Inc. and PyData Development Team
All rights reserved.

Reply:
I just put it there as my own reference for later use. Will definitely list the copyright

Reply:
Yes just here at the top?